source/reference/api/start.txt

```diff
@@ -151,12 +232,133 @@ Request
 ``RUNNING`` state.
+
+Pre-Split Chunks
+~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.1
+
+When ``mongosync`` syncs to a sharded cluster, it pre-splits chunks for
+sharded collections on the destination cluster.  This is supported in the
+following configurations:
+
+* Sync from a replica set to a sharded cluster.
+
+* Sync between sharded clusters that differ in the number of shards.
+
+
+Shard Replica Sets 
+~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.1
+
+Sync from a replica set to a sharded cluster requires the 
+``sharding`` option. This option configures how ``mongosync`` shards
+collections.
+
+The ``sharding.shardEntries`` array specifies the collections to shard.
+Collections that are not listed in this array replicate as unsharded.
+
+Supporting Indexes
+~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.1
+
+``mongosync`` syncs indexes from the source cluster to the destination
+cluster.  But, when syncing from a replica set to a sharded cluster,
+``mongosync`` may require the an additional index to support the shard key,
+which may not exist on the source cluster.
+
+``mongosync`` can create supporting indexes for sharded collections during sync.
+This is done by setting the ``sharding.createSupportingIndexes`` option.
+
+When ``sharding.createSupportingIndexes`` is ``false`` (the default): 
+
+* Each shard key you provide for the ``sharding.shardEntries`` option
+  must have an existing index on the source cluster.
+
+* One of the indexes used for the shard key must have simple collation if the 
+  collection uses any other collation.
+
+* To use a unique index in the shard key, you must specify its uniqueness when
+  you create the index on the source cluster. 
+
+* Unique indexes on the source cluster that are incompatible with the requested
+  shard key on the destination cluster, such as a unique index on the source
+  that does not contain the requested shard key as a prefix on the destination,
+  can cause ``mongosync`` to fail.
+
+When ``sharding.createSupportingIndexes`` is ``true``: 
+
+* If the supporting indexes exist on the source cluster, ``mongosync`` 
+  syncs the indexes to the destination cluster and uses them 
+  as the shard key.
+
+* If the supporting indexes don't exist, ``mongosync`` creates them on the
+  destination cluster.
+
+The ``sharding.createSupportingIndexes`` option affects all collections sharded
+during sync.
+
+Rename During Sync
+~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.1
+
+Collections listed in the ``sharding.shardEntries`` array 
+when synced from the replica set to a sharded cluster 
+become sharded collections on the destination cluster.
+
+Renaming a collection (such as with the :dbcommand:`renameCollection` command)
+on the source cluster after calling ``start`` but before ``mongosync`` begins 
+to copy the collection can block the collection from sharding on the destination.
+
+.. note:: 
+
+   Renaming collections to use a different database while ``mongosync``
+   is running is not supported
```

Comment:
I would add "when syncing from a replica set to a sharded cluster" in the note too.

Reply:
Fixed.